components/pkg-export-kubernetes/src/main.rs

```diff
@@ -95,65 +154,138 @@ fn start(_ui: &mut UI) -> result::Result<(), String> {
+    let mut out = r.lines().filter(|l| *l != "").collect::<Vec<_>>().join(
+        "\n",
+    ) + "\n";
 
-            match write.write(out.as_bytes()) {
-                Ok(_) => Ok(()),
-                Err(e) => Err(format!("{}", e)),
+    if let Some(binds) = matches.values_of("BIND") {
+        for bind in binds {
+            let split: Vec<&str> = bind.split(":").collect();
+            if split.len() < 3 {
+                return Err(Error::InvalidBindSpec(bind.to_string()).into());
             }
-        },
 
-        Err(e) => Err(format!("{}", e)),
+            let json = json!({
+                "name": split[0],
+                "service": split[1],
+                "group": split[2],
+            });
+
+            out += &Handlebars::new().template_render(BINDFILE, &json)?;
+        }
     }
+
+    write.write(out.as_bytes())?;
+
+    Ok(())
 }
 
 fn cli<'a, 'b>() -> App<'a, 'b> {
     let name: &str = &*PROGRAM_NAME;
-    clap_app!((name) =>
-        (about: "Creates a Kubernetes manifest for a Habitat package. Habitat \
-                 operator must be deployed within the Kubernetes cluster to \
-                 intercept the created objects.")
-        (version: VERSION)
-        (author: "\nAuthors: The Habitat Maintainers <humans@habitat.sh>\n\n")
-        (@arg IMAGE: --("image") +takes_value
-            "Image of the Habitat service exported as a Docker image")
-        (@arg OUTPUT: -o --("output") +takes_value
-            "Name of manifest file to create. Pass '-' for stdout (default: -)")
-        (@arg COUNT: --("count") +takes_value {valid_natural_number}
-            "Count is the number of desired instances")
-        (@arg TOPOLOGY: -t --("service-topology") +takes_value
-            "A topology describes the intended relationship between peers \
-             within a service group. Specify either standalone or leader \
-             topology (default: standalone)")
-        (@arg GROUP: -g --("service-group") +takes_value
-            "group is a logical grouping of services with the same package and \
-             topology type connected together in a ring (default: default)")
-        (@arg CONFIG_SECRET_NAME: -n --("config-secret-name") +takes_value
-            "name of the Kubernetes Secret containing the config file - \
-             user.toml - that the user has previously created. Habitat will \
-             use it for initial configuration of the service")
-        (@arg RING_SECRET_NAME: -r --("ring-secret-name") +takes_value
-             "name of the Kubernetes Secret that contains the ring key, which \
-              encrypts the communication between Habitat supervisors")
-        (@arg PKG_IDENT: +required
-            "Habitat package identifier (ex: acme/redis)")
-    )
+    let about = "Creates a docker image and Kubernetes manifest for a Habitat package. Habitat \
+                 operator must be deployed within the Kubernetes cluster to intercept the created \
```

Comment:
Not to intercept, but rather to create a deployment and other resources.

Reply:
hmmm.. how about "facilitate"?